packages/expo-localization/src/ExpoLocalization.web.js

```diff
@@ -20,7 +20,9 @@ export default {
       navigator.browserLanguage ||
       navigator.userLanguage ||
       this.locales[0];
-    return locale;
+
+    /* The native format is en-US */
```

Comment:
```suggestion
     // The native format is en-US
```

nit: use // for "internal" comments and `/** */` for "public" doc blocks